### DISCUSSION TITLE
How is the location of an SWA resource mapped into regions?

### DISCUSSION THREAD

**Discussion by leevi-sa** [Azure/static-web-apps#1249] [Q&A]
Or how should we intercept this location?

In our case, based on the data governance policies, we hope all our resources can be placed in Australia regions physically. So, it seems problematic for us if we chose either US, Europe or East Asia as a "location".

Does any of you have step on the same concerns? Or do you have any suggestions for this problem? Thank you.

**Comment by superhp** [comment]
We should separate SWA frontend (static contents) and backend (API, SSR). 
The frontend part is deployed to several locations around the globe. This is one of SWA benefits comparing to other services. As far as I know, it is not possible to limit this, at least for now. But do you have any restrictions for frontend deployment location? If so, you should probably use other service like WebApp. 
When you select SWA region, it is applicable only for the backend parts like API and SSR. If locations for SWA backend are not sufficient for you and SSR is not needed, you can deploy your backend to WebApp or other instance which has more locations, and then point it from SWA.

**Reply by leevi-sa** [reply]
Hi @superhp, Thank you for the valuable information. 
For the frontend part, we have concern mostly based on our general data residency policy of our organisation for the frontend code (As the tech team, we don't think there is any secrecy). After some internal discussions, we decided to use West US 2 option (we are an AU org).
For the backend part, we already using the independent Function App + Bring-your-own-api pattern, so it's not a problem.
